lstchain/reco/r0_to_dl1.py

```diff
@@ -571,10 +571,10 @@ def r0_to_dl1(
                     r1.waveform = r1.waveform[r1.selected_gain_channel, PIXEL_INDEX][np.newaxis, :, :] # Add the "channel" axis, all ctapipe wfs are 3D now
 
                     event.calibration.tel[tel_id].dl1.time_shift = \
-                        event.calibration.tel[tel_id].dl1.time_shift[r1.selected_gain_channel, PIXEL_INDEX]
+                        event.calibration.tel[tel_id].dl1.time_shift[:, PIXEL_INDEX]
```

Comment:
these lines can be removed completely now, right? They are no-ops

Reply:
Right, I was assuming there was a pixel reordering but that is not the case.